app/extensions.js

```diff
@@ -471,6 +531,17 @@ module.exports.init = () => {
     extensionActions.extensionDisabled(config.torrentExtensionId)
   }
 
+  // ipfsExtension
+  const isIPFSEnabled = getSetting(settings.IPFS_ENABLED)
+  console.log('IPFS: is extension enabled', isIPFSEnabled, config.ipfsExtensionId, getExtensionsPath('ipfs'))
```

Comment:
This prints: 
`IPFS: is extension enabled true interplanetaryfilesystemuniverse /Users/koruza/code/brave-browser-laptop/app/extensions/ipfs`